### ISSUE TITLE
Implement support for hostaliases 

### ISSUE THREAD

**Issue by rmacian** [terraform-providers/terraform-provider-kubernetes#365]
Add support for hostaliases as described in kubernetes pods

https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/

### Terraform Version
terraform --version
Terraform v0.11.13
+ provider.kubernetes v1.5.2.

### Affected Resource(s)
Please list the resources as a list, for example:
- deployment
- pod
- daemonset

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
What should have happened?

### Actual Behavior
What actually happened?

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Important Factoids
Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? Custom version of OpenStack? Tight ACLs?

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
https://github.com/terraform-providers/terraform-provider-kubernetes/pull/367

**Comment by alexsomesan** [comment]
Implemented via #367

**Comment by rmacian** [comment]
Which release will include this PR ? date expected ?
